flocker/volume/service.py

```diff
@@ -447,10 +474,12 @@ def _create_volume_service(cls, stderr, reactor, options):
 
         :return: The started ``VolumeService``.
         """
+        # Assign an OpenstackStoragePool here?
         pool = StoragePool(reactor, options["pool"],
                            FilePath(options["mountpoint"]))
         service = cls._service_factory(
             config_path=options["config"], pool=pool, reactor=reactor)
+        pool.volume_service = service
```

Comment:
This was a hack, so that we had (indirect) access to the node_id from the StoragePool. Do we still need this? If so, we should discuss.